tests/timeseries/_formula_engine/test_formula_composition.py

```diff
@@ -171,6 +171,61 @@ async def test_formula_composition_missing_bat(self, mocker: MockerFixture) -> N
 
         assert count == 10
 
+    async def test_formula_composition_min_max(self, mocker: MockerFixture) -> None:
+        """Test the composition of formulas with min/max values."""
+        mockgrid = MockMicrogrid(grid_side_meter=True)
+        await mockgrid.start(mocker)
+
+        logical_meter = microgrid.logical_meter()
+        engine_min = logical_meter.grid_power.min(Power.from_watts(0)).build(
```

Comment:
You might want to use the quantity zero constructor from SDK v0.23.0, `Power.zero()` instead